irohad/validation/impl/stateful_validator_impl.hpp

```diff
@@ -17,7 +17,10 @@
 #ifndef IROHA_STATEFUL_VALIDATIOR_IMPL_HPP
 #define IROHA_STATEFUL_VALIDATIOR_IMPL_HPP
 
+#include "backend/protobuf/transaction.hpp"
+#include "model/converters/pb_transaction_factory.hpp"
```

Comment:
It does not seem that these newly added (backend, model, validators) headers are required here.